codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -262,9 +262,6 @@ public void closeStream(final Http2Stream stream, ChannelFuture future) {
         future.addListener(new ChannelFutureListener() {
           @Override
           public void operationComplete(ChannelFuture future) throws Exception {
-            // Deactivate this stream.
-            connection().deactivate(stream);
```

Comment:
@Scottmitch I'm not sure what the original intent was here, but the initial call to `stream.close()` will deactivate the stream anyway so AFAICT the call to `connection().deactivate(stream)` is a no-op.